scene/3d/physics_body.cpp

```diff
@@ -915,6 +915,13 @@ String RigidBody::get_configuration_warning() const {
 
 	String warning = CollisionObject::get_configuration_warning();
 
+	if (get_scale() != Vector3(1.0, 1.0, 1.0)) {
```

Comment:
While I haven't tested this myself, `get_scale()` probably does not produce exact values for the scale in non-trivial cases. It might be better to do this instead:

    if (!get_scale().is_equal_approx(Vector3(1.0, 1.0, 1.0))) {